webapp/src/ts/services/message-contact.service.ts

```diff
@@ -71,9 +71,9 @@ export class MessageContactService {
   }
 
   isRelevantChange(change, conversation?) {
-    return (change.doc && change.doc.kujua_message)
-      || (change.doc && change.doc.sms_message)
-      || change.deleted
-      || (conversation && conversation.messages && conversation.messages.find(message => message.doc._id === change.id));
+    return (change.doc && change.doc.kujua_message) ||
```

Comment:
😅 Either way works, but usually I understand better when I read code and the operator is in front, like "prefixing" that the following lines are still part of the condition and not regular statements (like fn, assignments, etc). 

Is this breaking a lint rule?

Reply:
I like them better at the and and, yea it was breaking the line length rule with the operator at the end :(

Reply:
The last line is shortened by 3 chars if the operator is on the previous line.